src/snowflake/snowpark/_internal/proto/update-from-local.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# This script is a clone of update-from-devvm. It performs the same actions, except it uses the local clone of the Snowflake repo.
+# Example usage:
+#   ./src/snowflake/snowpark/_internal/proto/update-from-local.sh ~/src/Snowflake
+
+# Check for args
+SRC_ROOT="$HOME/src/Snowflake"
+if [ -z "$1" ]
+  then
+    echo "No source directory provided, defaulting to \"$SRC_ROOT\""
+elif [ -n "$1" ]
+  then
+    echo "Using source root \"$1\""
+    SRC_ROOT=$1
+fi
+set -euxo pipefail
+
+SCRIPT_DIR=$(dirname "$0")
+
+# Step 1: Build the python proto file from scratch via bazel
+pushd $SRC_ROOT/Snowpark
```

Comment:
```pushd ... > /dev/null && \
./make_ast.sh && popd
```
else, when make_ast.sh fails you're stuck in the wrong dir...

Reply:
Isn't it the opposite? We want to popd in all cases, not just when make_ast succeeds.